test/models/artefact_test.rb

```diff
@@ -231,6 +231,19 @@ class ArtefactTest < ActiveSupport::TestCase
     end
   end
 
+  should "validate redirect_url" do
+    artefact = FactoryGirl.create(:artefact)
+
+    artefact.redirect_url = "foobar"
+    refute artefact.valid?
+
+    artefact.redirect_url = "/foobar"
+    assert artefact.valid?
+
+    artefact.redirect_url = "http://foo.bar/"
+    assert artefact.valid?
```

Comment:
Just discussed it in the team - going to prevent absolute URLs and check the volume of support requests to add absolute URLs via router data.